schemas/x509.yml

```
@@ -0,0 +1,193 @@
+
+    - name: issuer.organizational_unit
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of organizational units (OU) of issuing certificate authority.
+      example: www.digicert.com
+
+    - name: issuer.organization
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of organizations (O) of issuing certificate authority.
+      example: DigiCert Inc
+
+    - name: issuer.locality
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of locality names (L)
+      example: Mountain View
+
+    - name: issuer.state_or_province
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of state or province names (ST, S, or P)
+      example: California
+
+    - name: issuer.country
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of country (C) codes
+      example: US
+
+    - name: signature_algorithm
+      level: extended
+      type: keyword
+      description: Identifier for certificate signature algorithm. Recommend using names found in Go Lang Crypto library (See https://github.com/golang/go/blob/go1.14/src/crypto/x509/x509.go#L337-L353).
+      example: SHA256-RSA
+
+    - name: not_before
+      level: extended
+      type: date
+      description: Time at which the certificate is first considered valid.
+      example: 2019-08-16T01:40:25Z
+
+    - name: not_after
+      level: extended
+      type: date
+      description: Time at which the certificate is no longer considered valid.
+      example: 2020-07-16T03:15:39Z
+
+    - name: subject.distinguished_name
+      level: extended
+      type: keyword
+      description: Distinguished name (DN) of the certificate subject entity.
+      example: C=US, ST=California, L=San Francisco, O=Fastly, Inc., CN=r2.shared.global.fastly.net
+
+    - name: subject.common_name
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of common names (CN) of subject.
+      example: r2.shared.global.fastly.net
+
+    - name: subject.organizational_unit
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of organizational units (OU) of subject.
+
+    - name: subject.organization
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of organizations (O) of subject.
+      example: Fastly, Inc.
+
+    - name: subject.locality
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of locality names (L)
+      example: San Francisco
+
+    - name: subject.state_or_province
+      level: extended
+      type: keyword
+      normalize:
+        - array
+      description: List of state or province names (ST, S, or P)
```

Perhaps we should spell out more explicitly here that "any of the `ST`, `S` or `P` entries are to be added to this array.

Current description is perfect for a `short` description. Then the new `description` could read like.

```suggestion
     short: List of state or province names (ST, S, or P)
     description: List of state or province names.
     
     All `ST`, `S` or `P` attributes should be added to the `x509.subject.state_or_province` array.
```